SPECS/golang/golang.spec

```diff
@@ -1,4 +1,6 @@
-%global bootstrap_compiler_version 1.19.12
+%global bootstrap_compiler_version_0 1.17.13
```

Comment:
It seems we have missed to release-bump:
- SPECS/application-gateway-kubernetes-ingress/application-gateway-kubernetes-ingress.spec
- SPECS/azcopy/azcopy.spec
- SPECS/blobfuse2/blobfuse2.spec
- SPECS/cf-cli/cf-cli.spec
- SPECS/cni-plugins/cni-plugins.spec
- SPECS/coredns/coredns.spec
- SPECS/csi-driver-lvm/csi-driver-lvm.spec
- SPECS/dcos-cli/dcos-cli.spec
- SPECS/etcd/etcd.spec
- SPECS/flannel/flannel.spec
- SPECS/gh/gh.spec
- SPECS/git-lfs/git-lfs.spec
- SPECS/gobject-introspection/gobject-introspection.spec
- SPECS/helm/helm.spec
- SPECS/influx-cli/influx-cli.spec
- SPECS/influxdb/influxdb.spec
- SPECS/jx/jx.spec
- SPECS/keda/keda.spec
- SPECS/KeysInUse-OpenSSL/KeysInUse-OpenSSL.spec
- SPECS/kube-vip-cloud-provider/kube-vip-cloud-provider.spec
- SPECS/local-path-provisioner/local-path-provisioner.spec
- SPECS/moby-buildx/moby-buildx.spec
- SPECS/moby-containerd-cc/moby-containerd-cc.spec
- SPECS/moby-containerd/moby-containerd.spec
- SPECS/moby-engine/moby-engine.spec
- SPECS/moby-runc/moby-runc.spec
- SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec
- SPECS/packer/packer.spec
- SPECS/skopeo/skopeo.spec
- SPECS/vitess/vitess.spec

And for Extended: 
- SPECS-EXTENDED/containernetworking-plugins/containernetworking-plugins.spec
- SPECS-EXTENDED/delve/delve.spec
- SPECS-EXTENDED/nmi/nmi.spec
- SPECS-EXTENDED/umoci/umoci.spec

This list may not be a 100% accurate as i've used very crude from of grep.